src/map/clif.c

```diff
@@ -17740,7 +17740,7 @@ void clif_parse_CashShopBuy(int fd, struct map_session_data *sd) {
 				if (!itemdb->isstackable2(data))
 					get_count = 1;
 
-				pc->paycash(sd, clif->cs.data[tab][j]->price * qty, kafra_pay);// [Ryuuzaki]
+				kafra_pay = pc->paycash(sd, clif->cs.data[tab][j]->price * qty, kafra_pay);// [Ryuuzaki] //changed Kafrapoints calculation. [Normynator]
```

Comment:
Unrelated to this pull request, but now I can no longer unsee it - does anyone know why does this function not do any sanity checks on the data received from the client, and doesn't check the error codes returned by `pc->paycash()`?